File: backend/src/main/java/com/ssafy/ganhoho/domain/member/dto/MemberDto.java
package com.ssafy.ganhoho.domain.member.dto;

import jakarta.persistence.*;
import lombok.*;

import java.time.LocalDateTime;

@Entity
@Table(name="member")
@Getter
@Builder
@AllArgsConstructor(access = AccessLevel.PROTECTED)
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class MemberDto {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long memberId;

    private String loginId;
    private String password;
    private String name;
    private String hospital;
    private String ward;
    @Column(insertable = false)
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
    private String appFcmToken;
    private String watchFcmToken;
}

File: ganhoho_app/app/src/main/java/com/ssafy/ganhoho/data/model/dto/member/MemberDTO.kt
package com.ssafy.ganhoho.data.model.dto.member

// 회원 정보 DTO
data class MemberDTO(
    val memberId: Long,    // 멤버 ID
    val loginId: String,   // 로그인 ID
    val name: String,      // 사용자 이름
    val hospital: String?, // 병원 (선택 사항)
    val ward: String?,     // 병동 (선택 사항)
//    val fcmToken: String?, // FCM 토큰 (선택 사항)
//    val deviceType: Int    // 기기 타입 (1: Android, 2: 워치)
)
